### DISCUSSION TITLE
A/B change reason

### DISCUSSION THREAD

**Discussion by enhany** [OE4T/meta-tegra#1271] [Q&A]
Hello,

in case of fail over to another slot (0 to 1 or 1 to 0), is there any way to get this information after boot?

```
root@jetson-xavier-nx-devkit-emmc:~# tegra-boot-control -s
Redundancy: enabled (full)
Current slot: 1
Slot 0: priority: 14, suffix: _a, retry_count: 7, successful: YES
Slot 1: priority: 15, suffix: _b, retry_count: 7, successful: YES
```

tegra-boot-control shows current slot, but slot can be changed in case of update or in case of fail over. How is possible to get the reason?

**Comment by madisongh** [comment]
This is something you'd have to design into your update system.
